sdk/go/pulumi/provider/provider.go

```diff
@@ -23,8 +23,8 @@ import (
 	"google.golang.org/grpc"
 )
 
-// This file relies on implementations in ../provider_linked.go that are made available in this package via
-// go:linkname.
+// This file relies on implementations in ../provider_linked.go that are made available in this
+// package via go:linkname.
```

Comment:
golangci-lint was flagging this as a bad `//go:linkname` directive because it was at the beginning of the line